emulator/src/test/java/com/scalar/client/tool/emulator/EmulatorTerminalTest.java

```diff
@@ -91,4 +91,109 @@ public void input_withSpacesForFlagInput_ShouldReturnProperParamsArrayProperly()
     // Assert
     Assert.assertArrayEquals(paramsArray, mockedParamsArray);
   }
+
+  @Test
+  public void get_withSpacesForInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get \"key with space\"";
+    String[] mockedParamsArray = { "get", "key with space" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void getJ_withSpacesForInput_ShouldReturnProperParamsArrayProperly() {
+    // Arrange
+    String input = "get \"key with space\"";
+    String[] mockedParamsArray = { "get", "key with space" };
+
+    // Action
+    String[] paramsArray = emulatorTerminal.paramsParser(input);
+
+    // Assert
+    Assert.assertArrayEquals(paramsArray, mockedParamsArray);
+  }
+
+  @Test
+  public void get_curlyBracket_ShouldReturnProperParamsArrayProperly() {
```

Comment:
I don' think `CurlyBracket` is a condition.

Reply:
Thank you for detailed review.
The test case method name updated.